Dockerfile

```diff
@@ -22,6 +22,8 @@ RUN apt-get update \
         make \
         wget \
         zlib1g-dev \
+        libreadline-dev \
+        libtinfo5 \
```

Comment:
Can we build ABC with `ABC_USE_NO_READLINE=1` to avoid these dependencies?

Otherwise, please keep the alphabetical ordering of dependencies (though `libboost` is out of order).

Reply:
Removed `libreadline-dev` and fixed the order.